<launch>
  <arg name="publish_raw" default="true"/>
  <arg name="publish_mask" default="true"/>
  <arg name="publish_markers" default="true"/>
  <arg name="publish_shear" default="true"/>
  <arg name="publish_depth" default="true"/>

  <node name="gelsight_stream" type="gelsight_stream_node" pkg="gelsight-ros">
    <rosparam command="load" file="$(find gelsight-ros)/config/gelsight.yml"/>

    <param name="publish_raw" value="$(arg publish_raw)"/>
    <param name="publish_mask" value="$(arg publish_mask)"/>
    <param name="publish_markers" value="$(arg publish_markers)"/>
    <param name="publish_depth" value="$(arg publish_depth)"/>
  </node>
</launch>
